_posts/2025-10-18-this-month-in-servo.md

```diff
@@ -0,0 +1,49 @@
+---
+layout:     post
+tags:       blog
+title:      "This month in Servo: and more!"
+date:       2025-10-18
+summary:    ao!! wrrrrao!!
+categories:
+---
+
+## Donations
+
+Thanks again for your generous support!
+We are now receiving **???? USD/month** (+??.?% over August) in recurring donations.
+
+Historically this has helped cover the cost of our [speedy](https://ci0.servo.org) [CI](https://ci1.servo.org) [servers](https://ci2.servo.org) and [Outreachy interns](https://www.outreachy.org/alums/2025-06/#:~:text=Servo).
+Thanks to your support, we’re now setting up **[two](https://ci3.servo.org) [new](https://ci4.servo.org) CI servers for benchmarking**, and **[funding the work of our long-time maintainer]({{ '/blog/2025/09/17/your-donations-at-work-funding-jdm/' | url }}) Josh Matthews** (@jdm), with a particular focus on helping more people contribute to Servo.
+
+Keep an eye out for [further CI improvements](https://github.com/servo/servo/issues/38141) in the coming months, including [**ten-minute WPT builds**](https://github.com/servo/ci-runners/issues/21), **macOS arm64 builds**, and **faster pull request checks**.
+
+Servo is also on [thanks.dev](https://thanks.dev), and already **?? GitHub users** (±? from August) that depend on Servo are sponsoring us there.
+If you use Servo libraries like [url](https://crates.io/crates/url/reverse_dependencies), [html5ever](https://crates.io/crates/html5ever/reverse_dependencies), [selectors](https://crates.io/crates/selectors/reverse_dependencies), or [cssparser](https://crates.io/crates/cssparser/reverse_dependencies), signing up for [thanks.dev](https://thanks.dev) could be a good way for you (or your employer) to give back to the community.
+
+<figure class="_fig" style="width: 100%; margin: 1em 0;"><div class="_flex" style="height: calc(1lh + 3em); flex-flow: column nowrap; text-align: left;">
+    <div style="position: relative; text-align: right;">
+        <div style="position: absolute; margin-left: calc(100% * 5552 / 10000); padding-left: 0.5em;"><strong>5552</strong> USD/month</div>
+        <div style="position: absolute; margin-left: calc(100% * 5552 / 10000); height: calc(1lh + 1.5em); border-left: 1px solid;"></div>
+        <div style="position: absolute; margin-left: calc(100% - 0.5em); height: calc(1lh + 1.5em); border-left: 1px solid;"></div>
+        <div style="padding-right: 1em;"><strong>10000</strong><!-- USD/month --></div>
+    </div>
+    <progress value="5552" max="10000" style="transform: scale(3); transform-origin: top left; width: calc(100% / 3);"></progress>
+</div></figure>
+
+As always, use of these funds will be decided transparently in the Technical Steering Committee.
```

Comment:
We can call out our new funding request process here, perhaps? Something like `To submit a proposal for funding, see our [funding request process](https://github.com/servo/project/blob/main/FUNDING_REQUEST.md).`

Reply:
Yes, we can also put a link to https://github.com/servo/project/issues/187 for transparency about how we're using the funds.

Reply:
done in 3ee88f6909af21f99638ad553416eb837d483195